CMakeLists.txt

```diff
@@ -253,6 +253,21 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
   find_package(LAPACKE REQUIRED)
   target_link_libraries(DDC INTERFACE ${LAPACKE_LIBRARIES})
   target_include_directories(DDC INTERFACE ${LAPACKE_INCLUDE_DIRS})
+
+  # Kokkos-kernels
+  if(NOT TARGET Kokkos::kokkoskernels)
+    find_package(KokkosKernels)
+    if(NOT KokkosKernels_FOUND)
+      set(KokkosKernels_ENABLE_ALL_COMPONENTS OFF)
+      set(KokkosKernels_ENABLE_COMPONENT_BLAS ON)
+      set(KokkosKernels_ENABLE_COMPONENT_BATCHED ON)
+      set(KokkosKernels_ENABLE_COMPONENT_LAPACK OFF)
+      set(KokkosKernels_ENABLE_TPL_BLAS OFF)
+      set(KokkosKernels_ENABLE_TPL_LAPACK OFF)
+      add_subdirectory(vendor/kokkos-kernels)
+    endif()
+  endif()
+  target_link_libraries(DDC INTERFACE Kokkos::kokkoskernels)
 endif()
```

Comment:
One last thing, we will need to handle this new dependency as Kokkos. We need to introduce an option `DDC_KokkosKernels_DEPENDENCY_POLICY`.

Reply:
addressed in [7ff91e0](https://github.com/CExA-project/ddc/pull/480/commits/7ff91e0aab724e982fdc0f8a06c6900c8799a553)